app/templates/index.hbs

```diff
@@ -43,7 +43,7 @@
       </div>
       <div class="lg:col-2">
         <h3 class="text-lg">Build pipeline</h3>
-        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, too— build your app for production with a single command.</p>
+        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, to build your app for production with a single command.</p>
```

Comment:
```suggestion
        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, too! Build your app for production with a single command.</p>
```

Reply:
yes, that's why I thought ` too—` might be a typo. 
after replacing   `too—` with  `too!` it looks fine.